app/views/comments/create.js.erb

```diff
@@ -0,0 +1,17 @@
+<% if @comment.errors.any? %>
+  $('#alert p').html("<%= escape_javascript(@comment.errors.full_messages.join('<br/>')) %>");
+  $('#alert').fadeIn();
+<% else %>
+  <% if @repository.number_of_comments == 1 %>
+    $(".comment_string").text(" Comment");
+  <% elsif @repository.number_of_comments == 2 %>
+    $(".comment_string").text(" Comments");
+  <% end %>
+  $(".number_of_comments").text("<%= escape_javascript(@repository.number_of_comments.to_s) %>");
+  $("<%= escape_javascript(render @comment) %>").appendTo(".container-full");
+  $('#alert p').html("New comment posted");
+  $('#alert').fadeIn();
```

Comment:
Note that the other branch of this if-else statement is also calling `$('#alert').fadeIn();`. Maybe you can just call this after the if-else statement, so it's valid for both branches.